api/swagger.json

```diff
@@ -0,0 +1,1304 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "This is the API specification for Chapter",
+    "version": "0.1.0",
+    "title": "Chapter",
+    "termsOfService": "https://www.freecodecamp.org/news/terms-of-service/",
+    "contact": {
+      "email": "chapter+dev@freecodecamp.org"
+    },
+    "license": {
+      "name": "BSD 3-Clause",
+      "url": "https://github.com/freeCodeCamp/chapter/blob/master/LICENSE"
+    }
+  },
+  "tags": [
+    {
+      "name": "chapter",
+      "description": "Everything about your Chapters",
+      "externalDocs": {
```

Comment:
Believe we should drop externalDocs links until we have that documentation in place?